Makefile

```diff
@@ -59,3 +62,10 @@ helm-docs: $(HELM_DOCS)
 ngc-helm-docs: download-ngc-helm-chart helm-docs
 
 branch-helm-docs: download-branch-helm-chart helm-docs
+
+.PHONY: generate-docs-versions-var
+generate-docs-versions-var:
+	cd hack && curl -sL ${BRANCH_RELEASE_YAML_URL} -o release.yaml
```

Comment:
can you copy this to a folder under $(CURDIR)/build/  like we do with helm charts ?

Reply:
Done